client/components/Layout/LayoutPubs.tsx

```diff
@@ -1,10 +1,12 @@
 import React from 'react';
 
 import { PubPreview } from 'components';
-import { Pub } from 'utils/types';
+import { Pub as BarePub } from 'utils/types';
 import { LayoutBlockPubs } from 'utils/layout/types';
 import { createReadingParamUrl } from 'client/utils/collections';
 
+type Pub = DefinitelyHas<BarePub, 'releases'>;
```

Comment:
The name `BarePub` kind of confused me at first. Do you think it'd be better to keep `Pub` and call `DefinitelyHas<Pub, 'releases'>` something like `PubWithReleases` or `ReleasedPub`?